content/file-and-directory-naming/javascript/_index.md

```diff
@@ -29,4 +29,4 @@ Students are expected to be on the same page when it comes to naming their files
     >src
```

Comment:
This looks a bit inconsistent. How do we usually describe directory hierarchies?